test/performance/utility/detail/bits_of_benchmark.cpp

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
The file should be called `seqan3/test/performance/std/bit_benchmark.hpp`, can you move it there?